src/Sharpliner/AzureDevOps/Model/Step.cs

```diff
@@ -70,6 +70,13 @@ public abstract record Step
     [DisallowNull]
     public ConditionedDictionary Env { get; init; } = [];
 
+    /// <summary>
+    /// Number of retries if the task fails.
+    /// Default is 0
+    /// </summary>
+    [YamlMember(Order = 230)]
+    public Conditioned<string>? RetryCountOnTaskFailure { get; init; }
```

Comment:
Why not `int`?

```suggestion
    public Conditioned<int>? RetryCountOnTaskFailure { get; init; }
```

Reply:
I am perfectly fine with having an int, in fact I would prefer it as it saves typing quotes, but the docs specific as a string https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-task?view=azure-pipelines

Reply:
I am perfectly fine with having an int, in fact I would prefer it as it saves typing quotes, but the docs specify it as a string https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-task?view=azure-pipelines

Reply:
Nice catch but I'd just say the docs are off (that was my experience in many places) because I see `string` in `timeoutInMinutes` too. So let's just make this an `int` for a better experience because we can well expect people will be putting in numbers